src/app/app.component.ts

```diff
@@ -9,11 +10,28 @@ import { Component, ViewEncapsulation } from '@angular/core';
 export class AppComponent {
   title = 'Workshop Angular';
   description = 'Angular Introduction';
+  showFiller = true;
+  isMenuVisible = true;
+  href='';
 
-  constructor() {
+  constructor(private router : Router) {
   }
   
   ngOnInit() {
+    this.href = this.router.url;
+    console.log(this.href);
+    /*TO DO
+    research how does this.router.url looks like*/
```

Comment:
You'll need to determine if the user is logged in and just then show the menu, if not on the login or register page.